PageEdit/PageEdit.js

```diff
@@ -0,0 +1,41 @@
+const linkToPortfolioPage = "../Portfolio/Portfolio.html";
+
+function redirectToPortfolioPage() {
+  window.location.href = linkToPortfolioPage;
+}
+
+  const addExperienceButton = document.getElementById("addExperienceButton");
+  const experienceForm = document.getElementById("experienceForm");
+  const saveExperienceButton = document.getElementById("saveExperienceButton");
+  const experienceEntries = document.getElementById("experienceEntries");
+
+  addExperienceButton.addEventListener("click", () => {
+    experienceForm.style.display = "flex";
+  });
+
+  saveExperienceButton.addEventListener("click", () => {
+    const companyName = document.getElementById("companyName").value;
+    const startDate = document.getElementById("startDate").value;
+    const endDate = document.getElementById("endDate").value;
+    const description = document.getElementById("description").value;
+
+    if (companyName === "" || startDate === "" || endDate === "" || description === "") {
+      alert("Please fill in all fields.");
+      return;
+  }
```

Comment:
dont need this, you are setting all the input fields as required. This is redundant.

Reply:
I removed the check for here. Thank you